### PR TITLE
Fix #3229. Add privacy policy notice when logging in.

### LINKED ISSUE #3229: Sirepo privacy policy notice
Add the below text to the email login page and the auth method selection page.

> By signing up for Sirepo you agree to Sirepo’s privacy policy and terms and conditions, and to receive informational and marketing communications from RadiaSoft. You may unsubscribe at any time.

### PR DESCRIPTION
**Pull request by e-carlin**

### CODE REVIEW

File: sirepo/package_data/static/html/login.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <div class="row">
     <div class="col-sm-6 col-sm-offset-3"><div class="text-center">
       <a style="margin: 0 10px" ng-repeat="m in login.authService.methods" data-ng-href="{{ m.url }}" class="btn btn-primary">{{ m.label }}</a>
+      <p class="help-block">By signing up for Sirepo you agree to Sirepo’s <a href="https://www.sirepo.com/en/privacy.html">privacy policy</a> and <a href="https://www.sirepo.com/en/terms.html">terms and conditions</a>, and to receive informational and marketing communications from RadiaSoft. You may unsubscribe at any time.</p>
```
Comment (robnagler) [inline sirepo/package_data/static/html/login.html:5]: apostrophe => single quote. no absolute uris.

File: sirepo/package_data/static/js/sirepo-components.js [hunk 2]
```diff
@@ -2593,19 +2593,24 @@ SIREPO.app.directive('emailLogin', function(requestSender, errorService) {
...
                   '<input type="text" class="form-control" data-ng-model="data.email" required/>',
                   '<div class="sr-input-warning" data-ng-show="showWarning">{{ warningText }}</div>',
                 '</div>',
               '</div>',
-              '<div class="row text-center" style="margin-top: 10px">',
-                 '<button data-ng-click="login()" class="btn btn-primary">Continue</button>',
+              '<div class="form-group">',
+              '<div class="col-sm-offset-2 col-sm-10 text-center">',
+                '<button data-ng-click="login()" class="btn btn-primary">Continue</button>',
+                '<p class="help-block">By signing up for Sirepo you agree to Sirepo’s <a href="https://www.sirepo.com/en/privacy.html">privacy policy</a> and <a href="https://www.sirepo.com/en/terms.html">terms and conditions</a>, and to receive informational and marketing communications from RadiaSoft. You may unsubscribe at any time.</p>',
```
Comment (robnagler) [inline sirepo/package_data/static/js/sirepo-components.js:2612]: apostrophe and uri.

**Review by robnagler — CHANGES_REQUESTED** [review]

File: sirepo/package_data/static/html/login.html (hunk 1 above)
Comment (e-carlin) [reply sirepo/package_data/static/html/login.html]: b152a032

File: sirepo/package_data/static/js/sirepo-components.js (hunk 2 above)
Comment (e-carlin) [reply sirepo/package_data/static/js/sirepo-components.js]: b152a032

**Comment by e-carlin** [comment]
@robnagler all comments addressed.

**Pull request merged**